picard/tagger.py

```diff
@@ -251,7 +251,8 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
         self.stopping = False
 
         # Load release version information
-        self.updatecheckmanager = UpdateCheckManager(parent=self.window)
+        if QtWidgets.QApplication.instance().autoupdate_enabled:
```

Comment:
Can't you simply use the `self.autoupdate_enabled` object here rather than the `instance()` object?